ArmaForces.Boderator/ArmaForces.Boderator.BotService/Discord/DependecyInjection/DiscordServiceBuilder.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace ArmaForces.Boderator.BotService.Discord.DependencyInjection
+{
+    internal class DiscordServiceBuilder : IDiscordServiceBuilder
+    {
+        private IServiceCollection Services { get; }
+
+        internal DiscordServiceBuilder(IServiceCollection services) => Services = services;
```

Comment:
Does not need to be internal. Please avoid internal methods.
```suggestion
        public DiscordServiceBuilder(IServiceCollection services) => Services = services;
```